capplets/common/gtkrc-utils.c

```diff
@@ -60,16 +60,20 @@ gchar* gtkrc_find_named(const gchar* name)
 
 	if (!path)
 	{
-		gchar* theme_dir = gtk_rc_get_theme_dir();
-		path = g_build_filename(theme_dir, name, subpath, NULL);
-		g_free(theme_dir);
-
-		if (!g_file_test(path, G_FILE_TEST_EXISTS))
-		{
-			g_free (path);
-			path = NULL;
-		}
-	}
+		const gchar * const * dirs = g_get_system_data_dirs();
+
+		if (dirs != NULL)
+			for (; !path && *dirs != NULL; ++dirs)
+                        {
```

Comment:
Mixed spaces and tabs. Looks good otherwise.

Reply:
> Mixed spaces and tabs. Looks good otherwise.

Fixed. Prefer tabs in this file, preserving the style already used.